bridgesync/types/types.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+type Unclaim struct {
+	GlobalIndex [32]byte `json:"global_index"`
```

Comment:
Maybe turn this to a `big.Int`. Its easier to use in tests and generally, because it has `.Bytes()` and `.String()` functions you can use in logging and when querying the `bridge syncer` `db`.